README.rst

```diff
@@ -128,36 +133,58 @@ Installation
     Changes`_ and `Testing`_ below if you just want to submit a bug fix or
     some new functionality.
 
-Assuming your system satisfies the installation requirements in the previous
-section, install SCons from this package by first populating the build/scons/
-subdirectory.  (For an easier way to install SCons, without having to populate
-this directory, use the scons-{version}.tar.gz or scons-{version}.zip
-package.)
+The preferred way to install SCons is through the Python installer, ``pip``
+(or equivalent alternatives, such as the Anaconda installer, ``conda``).
```

Comment:
in a virtualenv..

Reply:
I don't think we should over-push the virtualenv thing.  Unless someone is developing a Python project, it doesn't add that much value to isolate things... a "system" type install of SCons works just fine for very many cases.  Now if they need to install Python packages to work with whatever it is, the story of course changes.